packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -140,34 +410,37 @@ const Dropdown = forwardRef(
         insideOverflowWithTransformContainer
       });
 
+      type BaseStyles = typeof baseStyles;
```

Comment:
Why is it defined here?

Reply:
there is a Dropdown.styles.js file.
as I understand this is not a part of the current migration but I still need to use types somehow
so I used typeof to get style types.
this should be removed after migrating that file.

<img width="532" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/123493087/07cdf947-629e-4872-9ced-9af98ac9f1d3">